### ISSUE TITLE
How can I display multiple objects at the same time using different rendering methods?

### ISSUE THREAD

**Issue by Yuto126** [SFraissTU/BA_PointCloud#30]
Hello,

I thank you of your great work.

I want to display some objects at the same time using different rendering methods to increase the clarify. 
Do you have any ideas about this?

Thank you.

**Comment by Yuto126** [comment]
I mean BigSceneDemo + applying different rendering method for each point cloud in the given directory.

**Comment by SFraissTU** [comment]
Hi!
Well, the easiest way would be to have several Dynamic Point Cloud Sets active at the same time, one for each rendering method. However I would not recommend this, as you would have two rendering systems at the same time that would not share the point budget or the cache.
The proper way to do it would be a deeper change of the code. I would sponatenously do it roughly like this:
* Associate each PointCloudMetaData object with a rendering method by adding a public attribute of type MeshConfiguration (e.g. "config").
* Add a public attribute of type MeshConfiguration in PointCloudLoader, so that you can choose the config for each point cloud in the editor.
* In PointCloudLoader::LoadHierarchy, after creating the PointCloudMetaData object, set the objects meshconfiguration to the selected configuration (e.g. by metaData.config = this.config). 
* In V2Renderer, find all usages of n.RemoveGameObjects(config), node.RemoveAllGameObjects(config) and n.CreateGameObjects(config, pcset.transform), and replace config with n.MetaData.config (or node.MetaData.config).

This solution is a bit sloppy but I think it should work. I didn't test it, so feel free to let me know if that worked for you.

**Comment by Yuto126** [comment]
Hello,

Thank you for your helpful answer!

I have implemented the "Load Directory" button to create a MeshConfiguration for the file, and assign it to the PointCloudLoader config that is created at the same time.
Is my interpretation correct?

I have attached the implemented image below.
It worked without error and had the expected functionality!

<img width="1280" alt="スクリーンショット 2021-01-19 15 43 03" src="https://user-images.githubusercontent.com/44626917/104997722-707b5c80-5a6d-11eb-85d6-3383819fd4e3.png">

Thank you again for everything you’ve done.
